Can't include CSpec as an external CMake project

CMake allows to include an external project with Git adding these lines:

```cmake
 set(EXTERNAL_PROJECT_DIR ${CMAKE_BINARY_DIR}/external) 
  
 include(ExternalProject) 
 ExternalProject_Add(cspecs 
     GIT_REPOSITORY https://github.com/mumuki/cspec 
     CMAKE_ARGS -DCMAKE_INSTALL_PREFIX=${EXTERNAL_PROJECT_DIR} 
 ) 
  
 include_directories(${EXTERNAL_PROJECT_DIR}/include) 
 link_directories(${EXTERNAL_PROJECT_DIR}/lib)

 link_libraries(cspecs)
```

But it fails because the installation doesn't include CSpec's header:

```log
[  6%] Creating directories for 'cspecs'
[ 12%] Performing download step (git clone) for 'cspecs'
-- Avoiding repeated git clone, stamp file is up to date: '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-stamp/cspecs-gitclone-lastrun.txt'
[ 18%] Performing update step for 'cspecs'
HEAD is now at 80569c0 Merge pull request #29 from mumuki/shrink-image
[ 25%] No patch step for 'cspecs'
[ 31%] Performing configure step for 'cspecs'
-- Configuring done
-- Generating done
-- Build files have been written to: /home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build
[ 37%] Performing build step for 'cspecs'
make[5]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
make[6]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
make[7]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
Consolidate compiler generated dependencies of target cspecs
make[7]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
[100%] Built target cspecs
make[6]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
make[5]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
[ 43%] Performing install step for 'cspecs'
make[5]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
make[6]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
make[7]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
make[7]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
[100%] Built target cspecs
make[6]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
Install the project...
-- Install configuration: ""
-- Up-to-date: /home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/external/lib/libcspecs.so
make[5]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build/cspecs-prefix/src/cspecs-build'
[ 50%] Completed 'cspecs'
make[4]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build'
[ 50%] Built target cspecs
make[4]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build'
make[4]: Leaving directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build'
make[4]: Entering directory '/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/build'
[ 56%] Building C object CMakeFiles/commons-unit-test.dir/test_bitarray.c.o
/home/aranieri/so-commons-library-install-from-zip/tests/unit-tests/test_bitarray.c:20:10: fatal error: cspecs/cspec.h: No such file or directory
   20 | #include <cspecs/cspec.h>
      |          ^~~~~~~~~~~~~~~~
```

--- a/.gitignore
+++ b/.gitignore
@@ -6,3 +6,5 @@ release
 test.c
 example.c
 cspec.wiki
+/.idea/
+/cmake-build-*/

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,8 @@ set(SOURCE_FILES
         cspecs/cspec.c)
 
 add_library(cspecs SHARED ${SOURCE_FILES})
+set_target_properties(cspecs PROPERTIES PUBLIC_HEADER cspecs/cspec.h)
 
 install(TARGETS cspecs
         LIBRARY DESTINATION lib
-        ARCHIVE DESTINATION lib)
+        PUBLIC_HEADER DESTINATION include/cspecs)

